neuroscout_cli/cli.py

```diff
@@ -0,0 +1,46 @@
+"""
+neuroscout
+
+Usage:
+    neuroscout run [first_level|group_level] [-i <install_dir>|-w <work_dir>|-o <out_dir>|-c|--jobs=<n>] <bundle_id>
+    neuroscout install [bundle|data] [-i <install_dir>] <bundle_id>
+    neuroscout ls <bundle_id>
+    neuroscout -h | --help
+    neuroscout --version
+
+Options:
+    -i <install_dir>        Directory to download data and remote files.
+    -w <work_dir>           Working directory.
+    -o <out_dir>            Output directory.
+    -c                      Stop on first crash.
+    --jobs=<n>              Number of parallel jobs [default: 1].
+    -h --help               Show this screen.
+    --version               Show version.
+
```

Comment:
Is it possible to define the actions (e.g. "ls") down here? Would be in addition to anything docopt parses